docs/blog/2019-06-21-profitable-agency-wordpress/index.md

```diff
@@ -0,0 +1,115 @@
+---
+title: "Why Gatsby Can Be More Profitable for Your Agency Than WordPress"
+date: 2019-06-21
+author: Hashim Warren
+tags:
+  - agencies
+  - agency
+  - clients
+  - developers
+  - gatsby
+  - marketing
+  - wordpress
+  - aws
+---
+
+Are you a WordPress Developer?
+
+If yes, then you can relate to this. Every off-year we're greeted by a sales pitch from some hot new tool that wants to take the website builder throne.
+
+You know what I'm talking about. Their conference slide decks always have the same structure:
+
+### SLIDE 1: WordPress is slow.
```

Comment:
I think this heading level got missed. I tried to make a change for this directly to your fork but couldn't quite get it to work!